.travis.yml

```diff
@@ -10,4 +10,7 @@ after_failure:
   - wget https://raw.githubusercontent.com/k3rn31p4nic/travis-ci-discord-webhook/master/send.sh
   - chmod +x send.sh
   - ./send.sh failure $WEBHOOK_URL
-script: pytest tests
\ No newline at end of file
+script: pytest tests
+branches:
+  only:
+  - development
```

Comment:
dont you want travis running on intermediary builds? or does this make it manual-trigger-only? 🤔

Reply:
No, this effectively solves the "two builds on PRs" problem. See this PR's checks.
This sets branch update builds to only run for `development`.

Other branches, when pushed, won't get a build. But does that really matter since they're getting a PR build anyways?

Reply:
true. i was thinking "build before requesting a PR" but i guess you can wait till a PR is requested for a build test :P